File: CS/t328461/Models/Role.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace t328461.Models
{
    public class Role
    {
        public Role()
        {
            RoleID = 0;
            RoleName = string.Empty;
        }

        public Role(int id, String name)
        {
            this.RoleID = id;
            this.RoleName = name;
        }

        [Key]
        public int RoleID { get; set; }

        [Required(ErrorMessage = "Role Name is required")]
        public string RoleName { get; set; }
    }
}

File: VB/t328461VB/Models/Role.vb
Imports System.ComponentModel.DataAnnotations

Public Class Role

    Public Sub New()
        RoleID = 0
        RoleName = String.Empty
    End Sub

    Public Sub New(ByVal id As Integer, ByVal name As String)
        Me.RoleID = id
        Me.RoleName = name
    End Sub

    <Key>
    Public Property RoleID() As Integer

    <Required(ErrorMessage:="Role Name is required")>
    Public Property RoleName() As String
End Class